examples/js/vr/HelioWebXRPolyfill.js

```diff
@@ -4,6 +4,10 @@
 
 if ( /(Helio)/g.test( navigator.userAgent ) && "xr" in navigator ) {
 
+	if ('isSessionSupported' in navigator.xr) {
+        return;
+    }
+
```

Comment:
It seems this change causes #17837 because return is used not in function.

Reply:
Thanks for heads up. Taking a look.